readme.md

```diff
@@ -18,7 +18,7 @@
 
 ## Description
 
-By utilizing a simple and minimal usage syntax, that requires a flat learning curve, taskbook enables you to effectively manage your tasks and notes across multiple boards from within your terminal. All data are written atomically to the storage in order to prevent corruptions, and are never shared with any third party entities. Deleted items are automatically archived and can be inspected or restored at any moment.
+By utilizing a simple and minimal usage syntax, that requires a flat learning curve, taskbook enables you to effectively manage your tasks and notes across multiple boards from within your terminal. All data is written automatically to the storage in order to prevent corruptions, and are never shared with any third party entities . Deleted items are automatically archived and can be inspected or restored at any moment.
```

Comment:
I believe the correct fix is:
`All data is written atomically to the storage in order to prevent corruptions, and is never shared with any third party entities.`

The is/are usage needs to be changed, but the use of "atomically" is correct in this context. If actions are committed in an atomic fashion they are done so at a granular level that affects as few other things as possible.

Reply:
If you consider `data` to be a plural of `datum` (like "children" is to "child"), "All data are written..." is the correct way of writing this statement.

This is a source of contention, so both words (are vs is) are correct: https://en.wikipedia.org/wiki/Data_(word)